components/cards/ParcelInfo.tsx

```diff
@@ -214,31 +214,6 @@ function ParcelInfo(props: ParcelInfoProps) {
     loadLicenseDiamond();
   }, [licenseDiamondAddress, sfFramework, paymentToken]);
 
-  React.useEffect(() => {
```

Comment:
Is this a duplicate of above? Must be from a bad merge.

Reply:
Yes it's doing the same things minus last few lines.
Maybe was left there on merge because it didn't conflict.